src/mmw/apps/core/templates/base.html

```diff
@@ -52,6 +52,11 @@
         </script>
         <script type="text/javascript">
             window.clientSettings = {{ client_settings | safe }};
+
+            if (window.clientSettings.itsi_embed) {
+                // Disable CSS 3D transforms in embed mode to fix screenshots
```

Comment:
Consider updating this comment and noting that this is a Leaflet-specific setting.